15-transformer.ipynb

```diff
@@ -2,16 +2,20 @@
  "cells": [
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "id": "U_mTDKQcmOxc"
+   },
```

Comment:
Remove.

Reply:
In the new commits, I only change the lines need to be changed from original file without executing any blocks. I think this one would only include the commits you need.